dist/vokal-ng-api.js

```diff
@@ -187,7 +187,7 @@ angular.module( "vokal.API", [ "vokal.Humps" ] )
         };
         apiConstruct.prototype.setKey = function ( key )
         {
-            this.globalHeaders.AUTHORIZATION = String( key );
+            this.globalHeaders.AUTHORIZATION = key ? String( key ) : "";
```

Comment:
What if someone wants to pass an object for some crazy reason that is probably a mistake? You want to check typeof === number || string?